web-api/migration-terraform/main/lambdas/migrations/0004-standing-pretrial-order-signatures.js

```diff
@@ -34,10 +34,11 @@ const migrateItems = async (items, documentClient) => {
               return res.Item;
```

Comment:
I thought that if we needed to update something, we'd need to write a new migration for that instead of changing a previously-run migration - is that not how the migrations work?

Reply:
We do, but sometimes we fix migration bugs and re-migrate.

Reply:
As long as you find that acceptable, @mmarcotte, I'll resolve this.